neo/Consensus/ConsensusService.cs

```diff
@@ -347,8 +487,12 @@ private void OnStart()
                     context.Deserialize(reader);
                 }
             }
-            if (context.BlockIndex == Blockchain.Singleton.Height + 1 && context.State.HasFlag(ConsensusState.CommitSent))
+            if (context.State.HasFlag(ConsensusState.CommitSent) && context.BlockIndex == Blockchain.Singleton.Height + 1)
+            {
                 CheckPreparations();
+                // Set timer, so we will resend the commit in case of a networking issue
+                ChangeTimer(TimeSpan.FromSeconds(Blockchain.SecondsPerBlock));
```

Comment:
Should it be moved into `CheckPreparations()`?

Reply:
Yeah, that is probably a good idea. Will double check now.

Reply:
Done.